src/patternfly/components/OptionsMenu/docs/code.md

```diff
@@ -6,7 +6,7 @@
 | -- | -- | -- |
```

Comment:
@mcoker @jgiardino now that the react implementation is complete should I create an issue to fill in the options menu docs? or do you know if this is still on hold?